doc/source/theory/dem/dem.rst

```diff
@@ -67,17 +67,23 @@ The normal and tangential overlaps are calculated as follow:
     \delta_n =& \:R_i + R_j - |\mathbf{x}_{j} - \mathbf{x}_{i}| \\
     \mathbf{\delta}_{ij}^{t,\text{new}} &= \mathbf{\delta}_{ij}^{t,\text{old}}+\mathbf{v}_{rt}dt
 
-
-The relative velocities calculated to allow updating the tangential overlap are described by:
+~~~~~~~~~~~~~~~~~~~~~
+Relative velocities
```

Comment:
I suggest you use the Title case capitalization (words are capitalized, except for minor words) as done for other subsections in documentation.